wire-ios-data-model/Source/MLS/SubconversationGroupIDRepository.swift

```diff
@@ -65,4 +68,20 @@ public final class SubconversationGroupIDRepository: SubconversationGroupIDRepos
         return storage[parentGroupID]?[type]
     }
 
+    // MARK: - Finding subgroup
+
+    public func findSubgroupTypeAndParentID(
+        for targetGroupID: MLSGroupID
+    ) -> (parentID: MLSGroupID, type: SubgroupType)? {
+
+        for (parentID, subgroups) in storage {
+            if let subgroup = subgroups.first(where: {
+                $1 == targetGroupID
```

Comment:
what's $1 here?

Reply:
`$1` is the subgroup ID

Reply:
what's $0? maybe we use explicit name, or is it me just unfamiliar with the subgroupType

Reply:
updated [4887f50](https://github.com/wireapp/wire-ios/pull/598/commits/4887f50436ba14a75839a99c7447b530469a9c68)